.github/workflows/documentation-update.yml

```diff
@@ -22,53 +22,40 @@ jobs:
           node-version: 16
       - name: Install dependencies
         run: yarn install --frozen-lockfile
-      - name: Generate Client Docs (Javascript)
+      - name: Generate Client Docs
         run: yarn run docs:client
-      - name: Generate tokenVoting Client Docs (Javascript)
+      - name: Generate TokenVoting Docs
         run: yarn run docs:tokenVoting
-      - name: Generate Address List Client Docs (Javascript)
+      - name: Generate AddresslistVoting Docs
         run: yarn run docs:addresslistVoting
-      - name: Generate Multisig Client Docs (Javascript)
+      - name: Generate Multisig Docs
         run: yarn run docs:multisig
-      - name: Get short commit hash
-        id: hash
-        run: echo "sha_short=$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
-      - name: Commit updated docs
-        id: commit
-        run: |
-          git fetch
-          git pull
-          git config --global user.name "Arabot-1"
-          git config --global user.email "arabot-1@users.noreply.github.com"
-          git add -A ./docs/
-          if [ $(git diff-index HEAD ./docs | wc -l) -gt 0 ]; then
-            git commit -am "Updates docs from commit $SHA_SHORT"
-            git push
-          fi
-        env:
-          SHA_SHORT: ${{ steps.hash.outputs.sha_short }}
```

Comment:
Why removing the step that commits the updated docs? Are they pushed to the DP instead of the repo itself?

Does this affect the examples somehow? Or just the deprecated `docs/` folder?